src/hooks/usePlayer.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react'
+import { TETROMINOS, randomTetromino } from '../tetrominos'
+import { STAGE_WIDTH } from '../gameHelper'
+import { checkCollision } from '../gameHelper';
+
+
+
+interface point {
+    x: number
+    y: number
+}
+
+interface IPlayer {
+    pos: point
+    tetromino: (string | number)[][],
```

Comment:
Create type for `tetromino` instead
Like
`type Tetromino = Array<Array<string | number>>`